scripts/autogenerate-script.md

```diff
@@ -0,0 +1,13 @@
+# Use cases supported by autogenerate.sh
+
+## Use cases:
+1. API Def url can be any git repo url for which user has access ex:
+- ./autogenerate.sh git@github.com:rdkcentral/rdk-halif-device_settings.git
```

Comment:
how do I auto-generate from a local directory that I've cloned? Did you add that?

Reply:
`./autogenerate.sh ./git/clone/rdk-halif-device_settings` sort of thing? Can I do that?

Reply:
do you have a test script for this that we can add... Just so run and get no errors out of, like the examples your listing here, can we just make it into a test script.

Reply:
> `./autogenerate.sh ./git/clone/rdk-halif-device_settings` sort of thing? Can I do that?

Yes, this is done now.

> do you have a test script for this that we can add... Just so run and get no errors out of, like the examples your listing here, can we just make it into a test script.

Yes, I have also added [test script](https://github.com/rdkcentral/ut-core/pull/123/files#diff-840f613eb29ad4e0a7cddf5a9a1102f2bb0e92c9f46403659db8780112765620).